.github/workflows/continuous-integration.yml

```diff
@@ -1,62 +1,51 @@
 name: "Continuous Integration"
 
 on:
-  workflow_dispatch:
+  push:
```

Comment:
Do you need to include `workflow_dispatch:`?

Reply:
@maxakropolis I have. I added it back at the top.